starters/features/leaflet-map/src/helpers/boundary-box.tsx

```diff
@@ -0,0 +1,11 @@
+import type { Map } from "leaflet";
+export const getBoundaryBox = (map: Map, log: boolean = false) => {
+  const northEast = map.getBounds().getNorthEast();
+  const southWest = map.getBounds().getSouthWest();
+
+  log &&
+    console.log(
+      `${southWest.lat},${southWest.lng},${northEast.lat},${northEast.lng}`,
```

Comment:
Remove console.log please

Reply:
Good morning,
I have looked at everything carefully, but I am seeing that I have left out details in the checks, let's see if I get it right now ;)

The log thing is already removed ;) I was doing tests and I missed it, I think the rest will be clean, I have already done several tests.

Thank you so much! I hope you like the functionality, I have ideas for more interesting things :)